### ISSUE TITLE
.gitignore not working on folder

### ISSUE THREAD

**Issue by neothemachine** [feross/standard#172]
I have `src/jspm_packages/` in my .gitignore but standard scans it anyway. The same also happens when I put `"ignore": ["src/jspm_packages/"]` in package.json under "standard". Only when I use `"ignore": ["src/jspm_packages/**"]` it works. I'd like to use .gitignore however. Am I doing something wrong here? I'm on Windows if that matters.

**Comment by feross** [comment]
This is supposed to work.

What version of `standard` are you using? Please ensure you are using the latest version. This issue was supposedly fixed in 4.1.1, as you can see in the discussion here: https://github.com/feross/standard/issues/125#issuecomment-110904019

Please run: `npm install standard@latest --save-dev` to ensure you have the latest version.

**Comment by neothemachine** [comment]
I use the latest 4.3.2, but just in case I have run the install command 
again.

**Comment by feross** [comment]
Are you sure you're not running a globally installed version that's out of date? You can update that with `npm install standard -g`.

I'm not sure what the issue may be, but if you figure something out, PR welcome!

**Comment by feross** [comment]
@neothemachine Do you have any updates?

**Comment by neothemachine** [comment]
Sorry for the delay. I just did some more tests (with 4.4.0 installed globally):

The following patterns are handled correctly in standard in .gitignore on my system (Windows):
- build.js
- src\build.js
- src\\build.js
- src\\\build.js (for whatever reason; this was just a test)
- src\jspm_packages\*
- src\\jspm_packages\\*

Those don't work:
- src/build.js
- src/jspm_packages/
- src/jspm_packages/*
- src\jspm_packages\
- src\\jspm_packages\\

**Comment by feross** [comment]
It looks like a regression was introduced in @mafintosh's bugfix PR from last week: https://github.com/feross/standard/pull/168

The `ignore` package which we use to handle `.gitignore` patterns does not handle windows style paths, so we have to convert windows paths to unix style before calling it. It turns out that the node.js `path` functions (`path.resolve`, `path.join`) always convert path separators to the system's style (`\` on windows, `/` on unix). So when the `path` functions were introduced before the call to `ignore`, we started passing windows style paths to that module!

I should have noticed this before merging. My bad!

Fixed in 4.4.1.

@neothemachine Thanks for the list of patterns – it was very helpful.
